concourse/verify-production-urls/script.bash

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -ex
+
+bucket=sandbox-unified-web-primary
+production_url=https://openstax.org
+
+work_dir=$(pwd)
+
+release_id=$(curl "${production_url}/rex/release.json" | jq .id -r)
+files=$(aws s3api list-objects --bucket "$bucket" --prefix "rex/releases/$release_id/books/" | jq -r '.Contents[] | .Key')
+
+while IFS= read -r full_path; do
+  book_path=$(grep -oP '/books.*' <<< "$full_path")
+  [ ! -f "$work_dir$book_path" ] && exit 1
```

Comment:
the task is going to drop you in a build directory that has subdirs for each input and output resource. you have an input called `release`, so you'll have a release folder with all the stuff in it. so from where you are the path will be like `release/books/..`